src/App.jsx

```diff
@@ -14,70 +16,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <h1 className="title is-flex is-align-items-center">
+        { !selectedGood
+          ? ('No goods selected')
+          : `${selectedGood} is selected`
+        }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+        {selectedGood
+          ? (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => {
+                setSelectedGood('');
+              }}
+            />
+          )
+          : ''
```

Comment:
don't see any sense to use ternary operator here, instead of that you can use 
{selectedGood && ( ... )}